pkg/client/monitoring/v1/types.go

```diff
@@ -367,7 +367,7 @@ type Endpoint struct {
 	BasicAuth *BasicAuth `json:"basicAuth,omitempty"`
 	// MetricRelabelConfigs to apply to samples before ingestion.
 	MetricRelabelConfigs []*RelabelConfig `json:"metricRelabelings,omitempty"`
-	//Optional ProxyURL
+	// Optional ProxyURL e.g. http://proxyserver:2195. Directs scrapes to proxy through this endpoint
```

Comment:
End sentence with a period. For it to be marked optional in the documentation this needs to be a pointer to a string (*string). Then you can also skip the “Optional” start in the comment.